index.js

```diff
@@ -42,12 +44,13 @@ module.exports = (context, options) => {
   };
 
   let presets = [
-    ['@babel/preset-env', envOpts],
+    esmodules ? '@babel/preset-modules' : ['@babel/preset-env', envOpts],
     '@babel/preset-react',
     '@babel/preset-flow',
   ];
 
   let plugins = [
+    '@babel/plugin-syntax-dynamic-import',
```

Comment:
This addition is necessary only with `@babel/preset-modules` since included with  `@babel/preset-env`, correct?

Reply:
Correct :)

Reply:
Should we add it in this case only then?

Reply:
Yup, I first added it with a condition, then though it was not a big deal. Will update.